src/Symfony/Component/Messenger/Middleware/Enhancers/ActivationMiddlewareDecorator.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Middleware\Enhancers;
```

Comment:
Is the `Enhancers` sub-namespace required? Nesting hurts discovery IMHO, better remove it?

Reply:
This was mainly echoing `Symfony\Component\Messenger\Transport\Enhancers` which are decorators. But of course I'm fine with removing this namespace.